### ISSUE TITLE
Run Query On Demand

### ISSUE THREAD

**Issue by phidias51** [firebase/polymerfire#82]
Typically in an SPA there are a number of views that aren't visible.  I want to trigger the query to run only when a specific view is visible, not at app startup.  Is there a way of doing this?  If so, can you document it?

**Comment by mbleigh** [comment]
There isn't currently, but I'm thinking of adding an `inactive` boolean flag that would allow you to control when a query/doc listener should and should not be running.

**Comment by mbleigh** [comment]
This has shipped as the `disabled` attribute flag

**Comment by phidias51** [comment]
I used the disabled flag like this:
```html
<firebase-auth id="auth" user="{{user}}" app-name="quikintro" provider="google" on-error="handleError">
		</firebase-auth>

		<firebase-query id="doc"
		  app-name="quikintro"
		  path="[[url]]"
		  data="{{model}}"
		  disabled="{{disabled}}"
		  log
		>
		</firebase-query>


		<template is="dom-repeat" items="{{model}}" as="meeting">
			<qi-meeting-card model="{{meeting}}"></qi-meeting-card>
		</template>
```
I tried setting the disabled flag to false, expecting that it would automatically execute the query, but that didn't happen.    I added observers for array push, and splice events, but there's nothing in the model.  When I use the same path in the firebase console, it works.

Ideally what should happen is that whenever some condition is true, the query should execute, and the model (which should be an array not an object) should then begin to grow as nodes are added to it.  The data in those nodes should then be displayed in the cards either using a "dom-repeat" template, or an iron-list.  I've tried this on two separate projects with the same results.
